scripts/build_cutout.py

```diff
@@ -92,28 +88,38 @@
 """
 
 import logging
+import atlite
+import geopandas as gpd
+import pandas as pd
 from _helpers import configure_logging
 
-import os
-import atlite
+
+logger = logging.getLogger(__name__)
 
 logger = logging.getLogger(__name__)
 
 if __name__ == "__main__":
     if 'snakemake' not in globals():
         from _helpers import mock_snakemake
-        snakemake = mock_snakemake('build_cutout', cutout='europe-2013-era5')
+        snakemake = mock_snakemake('build_cutout', cutout='base')
```

Comment:
I see `base` is used here again. Any particular reason?
With `base` being commented out in `config.default.yaml` this would not work.
The keeping the `mock_snakemake` on one of the productive cutouts like `europe-2013-era5` seems more reasonable to me.